sw/device/silicon_creator/rom_ext/rom_ext_immutable.c

```diff
@@ -0,0 +1,19 @@
+// Copyright lowRISC contributors (OpenTitan project).
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/base/macros.h"
+#include "sw/device/silicon_creator/lib/dbg_print.h"
+
+#include "hw/top_earlgrey/sw/autogen/top_earlgrey_memory.h"
+
+/**
+ * Immutable ROM_EXT code section.
+ *
+ * This is a weak symbol to facilitate E2E testing.
```

Comment:
I don't see an `OT_WEAK` on this symbol.

Reply:
Fixed.